classes/app.rb

```diff
@@ -0,0 +1,146 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
```

Comment:
If I give a wrong date it is not rising an error. Instead it is breaking the app:

![Screen Shot 2022-08-27 at 05 42 47](https://user-images.githubusercontent.com/69011963/187007775-8048b495-4cc9-45f3-8d87-a1094e0874eb.png)

- [ ] Kindly make sure this error must be fixed to make your code approve!